### ISSUE TITLE
Can connect stripe app locally to my account all of a sudden was working before.

### ISSUE THREAD

**Issue by wispyco** [stripe/stripe-apps#987]
**Describe the bug**
I have built a stripe app, it compiles when I run `stripe apps start`
When I hit enter to login it allows me to choose the account I am logged in with earlier with stripe login but it just hangs and it has been doing this all day.

**To Reproduce**
1. Go to terminal
2. Type stripe apps start
3. Hit Enter
4. Connect to Account

**Expected behavior**
I would be logged into my stripe account with my stripe app loaded

**Screenshots**
![Screenshot 2024-09-13 at 7 13 11 PM](https://github.com/user-attachments/assets/5f5a1212-7a8e-45b0-aaf0-28dc0b874012)

![Screenshot 2024-09-13 at 7 14 00 PM](https://github.com/user-attachments/assets/8457e9f2-52d3-4959-bcee-f7c6b99ab580)

**Desktop (please complete the following information):**
 - OS: Mac OS Sonoma 14.3
 - Browser: Arc
 - Version: 1.60.0
 - Viewport dimensions: 1920 x 1080

**App (please complete the following information):**
 - App Id: com.example.catly-2
 - @stripe/ui-extension-sdk package version: ^8.9.1

**Additional context**
No

**Comment by bensontrent** [comment]
Do you have multiple stripe companies? And, have you run `stripe login`? You may need to run `stripe login` in your terminal before you run `stripe apps start`

You can only be logged into one company at a time from the CLI; you can only preview your app at one company at a time.  

To preview your app in multiple companies as a test app, run `stripe apps publish.`  Don't worry—this step doesn't make your app public, but your app will now have a private test installation link that can be used to install it in multiple companies as a test app.

In the screenshot you provided, you should be able to use the dropdown to select the company you are logged into:

Once you select the company you're logged into (through `stripe login`) the continue button will be enabled and you'll be able to preview your app.

![image](https://github.com/user-attachments/assets/2331c55f-fab3-4c0b-bdd4-ad7bfc101408)

**Comment by wispyco** [comment]
So I did that before and again as requested and it returned this

Done! The Stripe CLI is configured for Wispy Company with account

And it still hangs

![Screenshot 2024-09-13 at 8 45 27 PM](https://github.com/user-attachments/assets/d16ffff7-3b48-4422-9898-2390cbf9caa9)

**Comment by bensontrent** [comment]
Yes, you'll need to visit stripe.com using the link as directed in your terminal when you run `stripe login` or `stripe apps start`.

For example, when you run `stripe login` you need to click on the `ctrl + click+ link in your CLI to complete the login process.  Similarly, when you run `stripe apps start` you'll need to click the link in your terminal to initiate the preview mode for your app.  You should only need to run `stripe logout` and `stripe login` when you need to change the Stripe company you're previewing your app in.

**Comment by wispyco** [comment]
Actually there is no link you have to press enter to enter dashboard and the press enter redirect does not work thats what hangs.

![Screenshot 2024-09-13 at 8 56 54 PM](https://github.com/user-attachments/assets/2d72f9b8-859d-4f00-b4b4-916577480b8a)

https://dashboard.stripe.com/apps/enable-preview?cliaccount=redacted

This is the url it redirects me to that hangs

**Comment by bensontrent** [comment]
Sorry, my bad, you're right.  You have to press Enter. There is no link.  And to confirm, you were able to login to your desired account correct?

**Comment by wispyco** [comment]
Yes I did select the right company, and hit continue but it still just hangs I have to visit stripe.com in a new tab, maybe its a caching issue for me but I cleared everything in my dev tools application data so not sure, I have a workaround just open new tab after hitting enter in terminal and go stripe.com it works. Thank you
